src/components/ebay-menu-button/template.marko

```diff
@@ -1,25 +1,38 @@
+<script marko-init>
+    var processHtmlAttributes = require("../../common/html-attributes");
+</script>
+
+<var isFake=(data.type === 'fake')/>
+<var baseClass=(isFake ? 'fake-menu-button' : 'menu-button')/>
+
 <span
-    class=data.menuClass
+    class=[
+        (isFake ? 'fake-menu-button' : 'menu-button'),
+        data.class
+    ]
     style=data.style
     w-bind
     w-on-expander-expand="handleExpand"
     w-on-expander-collapse="handleCollapse"
-    ${data.htmlAttributes}>
+    ${processHtmlAttributes(data)}>
     <ebay-button
         w-id="button"
-        class=data.buttonClass
+        class=[
+            "${baseClass}__button",
+            (data.borderless && 'expand-btn--borderless')
+        ]
         variant="expand"
         size=data.size
         priority=data.priority
-        no-text=data.noText
-        aria-expanded=String(data.expanded)
+        no-text=(!data.text && !data.icon)
+        aria-expanded="false"
         aria-haspopup="true"
         aria-label=data.a11yText
         disabled=data.disabled
         w-on-button-escape="handleButtonEscape">
-        <span class=['expand-btn__cell', (data.customLabel && 'menu-button__control--custom-label')]>
-            <span if(data.customLabel) w-preserve-body>
-                <span w-body=data.customLabel.renderBody body-only-if(true)/>
+        <span class=['expand-btn__cell', (data.label && 'menu-button__control--custom-label')]>
+            <span if(data.label) w-preserve-body>
+                <span w-body=data.label.renderBody body-only-if(true)/>
```

Comment:
is `body-only-if(true)` a trick to render body content without the container?

Reply:
@msendlakowski that is correct. Oh Marko 3...